cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+def: "From the distribution of precursor intensities, the sigma value" [PSI:QC]
+comment: The intensity distribution of the precursors informs about the dynamic range of the acquisition.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+has_units: STATO:0000237 ! standard deviation
+
+[Term]
+id: QC:40000
+name: Precursor intensity distribution outliers (<Q1-1.5*IQR, >Q3+1.5*IQR)
+def: "From the distribution of precursor intensities, the list of outliers below of Q1-1.5*IQR and above Q3+1.5*IQR" [PSI:QC]
+comment: The intensity distribution of the precursors informs about the dynamic range of the acquisition.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000081 ! Outliers below
+has_column: QC:4000082 ! Outliers above
+
+[Term]
+id: QC:40000
+name: MS1 signal jump (10x) count
+def: "The count of MS1 signal jump by a factor of ten or more (10x) between two subsequent scans" [PSI:QC]
+comment: An unusual high count of signal jumps or falls can indicate ESI stability issues.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-1A"
+
+[Term]
+id: QC:40000
+name: MS1 signal fall (10x) count
+def: "The count of MS1 signal decline by a factor of ten or more (10x) between two subsequent scans" [PSI:QC]
+comment: An unusual high count of signal jumps or falls can indicate ESI stability issues.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-1B"
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 1+ over 2+
+def: Ratio of 1+ peptide count  over 2+ peptide count  in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A" NARROW []
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 3+ over 2+
+def: Ratio of 3+ peptide count over 2+ peptide count in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B" NARROW []
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 4+ over 2+
+def: Ratio of 4+ peptide count over 2+ peptide count in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synoym: "IS-3C" NARROW []
+
+[Term]
+id: QC:40000
+name: Mean charge in identified spectra
+def: Mean charge in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Median charge in identified spectra
+def: Median charge in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Charged spectra ratio +1 over +2
+def: Ratio of 1+ spectra count over 2+ spectra count in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A"  RELATED []
```

Comment:
add this to a 
```remark:
    synonyms can be 'RELATED` (the metric is similar, but not the same thing) or 'NARROW' (identical interpretation of numbers)
```
section on top of this .obo... so people know how to read this

Reply:
:+1: